spec/directives/for.hrx

```diff
@@ -149,6 +149,86 @@ a {
   i: 4;
 }
 
+<===>
+================================================================================
+<===> unit/same/input.scss
+a {
+  @for $i from 1px through 5px {b: $i;}
+}
+
+<===> unit/same/output.css
+a {
+  b: 1px;
+  b: 2px;
+  b: 3px;
+  b: 4px;
+  b: 5px;
+}
+
+<===>
+================================================================================
+<===> unit/to_unitless/options.yml
+---
+:todo:
+  - libsass
```

Comment:
the libsass behavior is to require the same unit rather than requiring compatible units. That's why these 3 specs are marked as TODO. I haven't created an issue on libsass for that given that it is deprecated.

Reply:
These should probably be `ignore_for` rather than `todo` since there's on expectation that LibSass will support them in the future.

Reply:
done.
Maybe the readme should be updated for its explanation of `ignore_for` as it is described as being used for deprecated features removed from modern implementation, not for modern features ignored for deprecated implementations. That's why I chose `todo` initially.